src/App.vue

```diff
@@ -63,6 +64,20 @@
     public get currencyFormat(): string {
       return currencies[this.snippetConfig.currency!];
     }
+
+    @Watch('snippetConfig.lang')
+    syncLang(lang: string): void {
+      this.$setLocale(lang);
```

Comment:
I think that if you change the `lang` you should change the `searchLang` too, as you are using the lang as searchLang in the `snippet-script.js` file.

Reply:
I understand from the `snippet-script.js` that `searchLang` is dependant on `lang` when `searchLang` doesn't have a set value, so maybe it doesn't apply here. At most I would change both if there's no `searchLang` URL parameter, but still don't see it as necessary. Up to you @tiborux